src/app/forms/suggestion-box/terra-suggestion-box.component.ts

```diff
@@ -255,9 +241,11 @@ export class TerraSuggestionBoxComponent implements OnInit, OnChanges
 
     public onChange()
     {
-        let searchString = this._selectedValue.caption;
+        let searchString:string = this._selectedValue.caption;
         this.toggleOpen = true;
 
+        this.changeSelectedSuggestionBoxEntryBySearchString(searchString);
```

Comment:
Bitte mache die neue Funktionalität optional verwendbar. Es ist nicht immer gewollt, irgendwelche Werte abschicken zu können.

Reply:
Die Komponente wurde ganz bewusst so entwickelt, dass sie einzig Werte nutzten kann die vorgeschlagen wurden.
D.h. wenn du der Meinung bist dies nicht optional für den Entwickler einschaltbar zu machen, müsste dies in eine neue Komponente, z.b. eine TypeAheadSearch.

Reply:
Garvin bitte hör auf meinen Code zu reviewen ich habe keine Lust hier meine Zeit zu verschwenden, danke.

Reply:
Geht es dir ein bisschen zu gut Jerome?